buildspec.yml

```diff
@@ -0,0 +1,40 @@
+version: 0.2
+
+env:
+  shell: bash
+phases:
+  install:
+    runtime-versions:
+      python: 3.10
+  pre_build:
+    commands:
+      - aws codeartifact login --tool pip --domain criticalstart --domain-owner 818476207984 --repository criticalstart_global
+      - # Add pip cache to speed up build time
+        # We need to put it in a directory accessible by the docker build context
+        # We'll attempt to make the directory in case the cache is currently empty
+        - mkdir -p /root/.cache/pip
+        - cp -R /root/.cache .cache
+        - pip install black
+        - pip install -r requirements-build.txt
+        - REPOSITORY_BASE=818476207984.dkr.ecr.us-west-2.amazonaws.com
+        - AWS_DEFAULT_REGION=us-west-2
+        - aws ecr get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin $REPOSITORY_BASE
+        - docker build -t app .
```

Comment:
This bit won't work.  If you check the README, there's a section on how to run tests.  This one doesn't have a DOCKERFILE.